fe/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -624,6 +632,14 @@ public Partition getPartition(String partitionName) {
         return nameToPartition.get(partitionName);
     }
 
+    public Partition getPartition(String partitionName, boolean isTempPartition) {
```

Comment:
If split into 2 methods, we have to write `if .. else ..` every time we call `getPartition()`, which is not convenient.

Reply:
>If split into 2 methods, we have to write if .. else .. every time we call getPartition(), which is not convenient.

split to two methods is in order to avoid ` if .. else .. `.

>But maybe I will change it when implementing loading into temp partition. It depends on how convenient it is to use in the loading process.

OK